pyfarm/agent/service.py

```diff
@@ -77,6 +77,8 @@ class Agent(object):
     itself with the master, starting the periodic task manager,
     and handling shutdown conditions.
     """
+    reactor = reactor
```

Comment:
Same reason for why #215 has:

```
It's not anymore, removing. Anytime you see something like this it's usually the reactor can be replaced in testing. In the case of this change I have not had a reason to use it for testing so far...
```
I'm removing the change in this PR for the same reason.